general/glossary.md

```diff
@@ -0,0 +1,52 @@
+# Glossary
+
+This documents aims at providing a common lexicon for terms used to refer to artifacts.
+
+### Motivation
+
+Throughout the documentation there are several terms which are often abused because we either have no alternative or there's not enough consensus on what's their canonical name.
+
+This leads to often ambiguous documentation because one term is used for different things or because we use different terms for the same artifact/object.
```

Comment:
I think you could frame this more positively. These two lines are basically saying: "the" documentation (which documentation, by the way?) is terrible, human beings are terrible, and everything is terrible.

Given that this is a _guidelines_ repo, I'd instead take a prescriptive tone; ie. tell people what they can do and why.

> Our technology space is filled with overloaded words like "package", "module", and "bundle". 
Despite the potential for confusion, we can make our documentation, APIs, and all forms of written and spoken communication clearer by carefully selecting and qualifying the terminology that we use.

Reply:
> "the" documentation (which documentation, by the way?) is terrible, human beings are terrible, and everything is terrible.

That's what I wanted to transmit in fact :joy: